pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -165,11 +165,16 @@ type CommonServiceBrokerSpec struct {
 //
 // ServiceClass allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
-//   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   spec.externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   spec.externalID - the value set to [Cluster]ServiceClass.Spec.ExternalID
+//   spec.serviceBrokerName - the value set to [Cluster]ServiceClass.Spec.[Cluster]ServiceBrokerName
 //
 // ServicePlan allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
-//   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   spec.externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   spec.externalID - the value set to [Cluster]ServiceClass.Spec.ExternalID
+//   spec.serviceClassName - the value set to [Cluster]ServiceClass.Spec.ServiceClassRef.Name
```

Comment:
please remove `spec.serviceClassName` from the comments here for plan and add spec.serviceBrokerName.

Please also fix `[Cluster]ServiceClass.Spec` for plan, it should be `[Cluster]PlanClass.Spec`